knimin/lib/data_access.py

```diff
@@ -443,11 +443,12 @@ def get_ag_barcode_details(self, barcodes):
             {barcode: {column: value}, ...}
         """
         sql = """SELECT DISTINCT barcode, *
-                 FROM ag_kit_barcodes
-                 JOIN ag_kit USING (ag_kit_id)
-                 FULL OUTER JOIN ag_login_surveys USING
-                    (survey_id, ag_login_id)
-                 JOIN ag_login USING (ag_login_id)
+                 FROM ag.ag_kit_barcodes
+                 JOIN ag.ag_kit USING (ag_kit_id)
+                 LEFT JOIN ag.source_barcodes_surveys USING (barcode)
+                 LEFT JOIN ag.ag_login_surveys USING (survey_id)
+                 LEFT JOIN ag.ag_login
+                 ON (ag.ag_kit.ag_login_id = ag.ag_login.ag_login_id)
```

Comment:
no, because there are three tables holding ag_login_id: ag_kit, ag_login_surveys and ag_login